Week1/Solution/1. Prime or Not.

```diff
@@ -0,0 +1,12 @@
+#write a python script to Enter any Number and check it is prime or not.
+n=int(input('Enter Any Number:: '))
+c=0 #it called count.it is used to define how many times the number is divided.
+i=1 
+for i in range(i,n+1):
```

Comment:
why you don't write for loop in given below way ?
for i in range(1,n+1):

Reply:
yes, we also execute the programe using this formula of for loop,
but that time i understood this for loop formula so i apply this.

we can execute one programe in lots of way.